src/datasources/fault-ds/client_delegate.js

```diff
@@ -0,0 +1,267 @@
+import {API, Client, Rest, DAO, Model} from '../../opennms'
+import _ from 'lodash';
+
+export class ClientDelegate {
+
+    constructor(settings, backendSrv, $q) {
+        this.type = settings.type;
+        this.url = settings.url;
+        this.name = settings.name;
+        this.backendSrv = backendSrv;
+        this.searchLimit = 1000;
```

Comment:
Can we make this unlimited by default? We currently have no way of indicating the results have been truncated, or providing an order by clause that would help control which items are being cut.